packages/components/organisms/f-form/README.md

```diff
@@ -73,17 +73,17 @@ export default {
 ### Props
 
 To add fields to the form and text to the form button a `formData` Object prop should be passed in. The `formData` Object should contain.
- * **formFields** - ` Array of objects. Object should include
+ * **formFields** - Array of objects. Object should include
     * **name** - The field name that data should be stored under
-    * **value** - The value of the field
+    * **value** - TThe value of the field. When not set, this defaults to ''.
     * **translations** - an object containing:
-        * **label** - the field label
-        * **validationMessages** - an object that can containin two types of validation messages:
+        * **label** - the field's displayed text label
+        * **validationMessages** - an object that can contain two types of validation messages:
```

Comment:
This is interesting, however it seems to be limiting to `required` and `invalid` without relation to the validation rule(s). We build something similar recently and had to build a field validity object, also observe validation order... 
It can easily get complicated. Happy to chat to you about that if that is the on the cards here.

Reply:
Yeah would be good to see what you've already built to see if there's anything that could be improved. 
At the moment it works out the validation message here https://github.com/justeat/fozzie-components/pull/1556/files#diff-fdc8192e0a3e68bb066e4093bc5964e0edec3662062fca586688c40fc5a9ca34R121 wheich is the same as is currently used in checkout